introduction_to_amazon_algorithms/object_detection_birds/object_detection_birds.ipynb

```diff
@@ -34,7 +34,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    !gunzip -c ./CUB_200_2011.tgz | tar xopf -
</pre><p>Why get rid of the <code>!rm CUB_200_2011.tgz</code>  that deletes the extra file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3007/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>because is expensive to download, download is skipped if it's already present.</p>